src/libcore/default.rs

```diff
@@ -84,10 +84,33 @@
 
 use marker::Sized;
 
-/// A trait for giving a type a useful default value.
+/// A trait for giving a type a useful default value. For more information, see
+/// [the module-level documentation][module].
```

Comment:
Yeah, so originally the thinking was like "docs shouldn't be repeated, they should only go in one place, what do?" But really, the module-level should provide a broad summary, the the struct itself should have all the gory details.